console/src/components/Services/Data/Function/Permission/Permission.js

```diff
@@ -78,14 +82,14 @@ class Permission extends React.Component {
           applicable to the data returned by this function
         </p>
         <div className={styles.commonBtn}>
-          <Link to={permissionTableUrl}>
+          <a href={permissionTableUrl}>
```

Comment:
This needs to be later changed back to <Link>.

Reply:
Link breaks this functionality due to a data requirement in permissions tab being not satisfied. I will fix this issue along with the permission data issue in a different PR

Reply:
Created an issue for the same #1567 and approving this PR.